src/main/java/org/fundacion/jala/converter/service/metadata/ExportTypeFile.java

```diff
@@ -5,26 +5,34 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
+ *
+ * @author Gustavo Huanca Alconz
  */
 package org.fundacion.jala.converter.service.metadata;
 
 import java.io.File;
 
+/**
+ * This class exports a metadata text file.
+ */
 public class ExportTypeFile {
     private File addressExportFile;
     private String nameFileComplete;
     private String nameFileExport;
     private String nameFileCompleteToExport;
 
-    public ExportTypeFile(final String nameFileComplete, final String nameExport, final TypeFileExport typeExport, final File addressFileExport) {
+    public ExportTypeFile(final String nameFileComplete,
+                          final String nameExport,
+                          final TypeFileExport typeExport,
+                          final File addressFileExport) {
         this.nameFileComplete = nameFileComplete;
         this.addressExportFile = addressFileExport;
         buildFileName(nameExport);
         buildFileType(typeExport);
     }
 
     /**
-     * Get parameter nameFileCompleteToExport
+     * Get parameter nameFileCompleteToExport.
      *
```

Comment:
It is "Gets" (third person)

Reply:
You are rigth, I have already corrected to third person